source/flow/tcp/tcp_flow.hpp

```diff
@@ -222,7 +221,7 @@ class TcpFlow : public IFlow,
     bool m_ecn_capable;
 
     std::uint32_t m_packets_in_flight;
-    std::uint32_t m_packets_acked;
+    SizeByte m_delivered;
```

Comment:
I think we should rename this to m_bytes_delivered, in the context of flow it's not clear if we speak about delivered packets or number of delivered bytes with current naming
```suggestion
    SizeByte m_delivered_bytes;
```

Reply:
Renamed to `m_delivered_data_size`